commands/build.go

```diff
@@ -231,6 +231,15 @@ func runBuild(dockerCli command.Cli, in buildOptions) (err error) {
 	if err != nil {
 		return err
 	}
+	for _, out := range outputs {
+		if out.Type == "docker" {
```

Comment:
this should use `driver.MultiPlatform`

Reply:
Or I guess because this is container driver with a Docker endpoint for exporting that endpoint should be checked similarly as `driver.MultiPlatform` is detected. If this gets complicated and doesn't work for regular multi-platform atm then it can be follow-up.

Reply:
I've reworked the patch to be handled in `toSolveOpt`, where we handle the exporter and multi-platform case like this today - this means it'll also work in bake.

I think you're right, we should check multi-platform capabilities in `toSolveOpt`, to see if we need to error with `"docker exporter does not currently support exporting manifest lists"` for either attestations or multi-platform?

But we can do that as a follow-up, it seems orthogonal to this - it makes sense for the behavior for attestations and multi-platform should be linked together in the same place (that both generate indexes).